src/styles/utils/_vars.scss

```diff
@@ -1 +1,11 @@
-$c-gray: #eee;
+$desktop-columns: 12;
+$tablet-columns: 6;
+$grid-gap-desktop: 20px;
+$grid-gap-tablet: 30px;
+$main-color: #333;
+$secondary-color: #344f10;
+$bcg-menu-contacts-color: #e4dfd3;
```

Comment:
It seems like there might be a typo in the variable name. Consider renaming $bcg-menu-contacts-color to $bg-menu-contacts-color.